app/pages/settings/email.jsx

```diff
@@ -251,6 +259,40 @@ class EmailSettingsPage extends React.Component {
               onChange={handleInputChange.bind(this.props.user)}
             />
           </AutoSave>
+          <div>
+            {(isEmailValid)
+              ? <div>
+                  <i className='fa fa-check-circle' style={{ color: '#51db72' }} />
```

Comment:
This icon will be missed by a screen reader. As an aside, icon fonts aren’t really used in web development any more, because they can cause accessibility problems.